kvmagent/kvmagent/test/unittest_tools/unittest_pypi_source/pypi_source_builder.sh

```
@@ -0,0 +1,149 @@
+        --target-path) BUILD_TARGET_PATH="$2"; shift 2;;
+        --requirement-output-path) PYPI_REQUIREMENT_OUTPUT_PATH="$2"; PYPI_REQUIREMENT_OUTPUT="true"; shift 2;;
+        -h|--help) call_help; exit 0;;
+        *) shift; break;;
+    esac
+done
+
+
+echo "PACKAGE_LIST_PATH=$PACKAGE_LIST_PATH"
+echo "BUILD_TARGET_PATH=$BUILD_TARGET_PATH"
+echo "PYPI_REQUIREMENT_OUTPUT_PATH=$PYPI_REQUIREMENT_OUTPUT_PATH"
+mkdir -p $(dirname $PYPI_REQUIREMENT_OUTPUT_PATH)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)